vcd/sample_vcd_test_config.json

```diff
@@ -90,6 +90,44 @@
     "uploadPieceSize": 5,
     "uploadProgress": true
   },
+  "testEnvBuild": {
+    "//": "This section is optional for running tests. It's only needed to build the test environment in a new vCD",
+
+    "// ": "Entities that will be created in an empty vCD use names defined above",
+
+    "// ": "org:              from vcd.org",
```

Comment:
This format kind'a strange for me, I want to uncomment and write down `org: myorg1`, etc.
Maybe it's only me

Reply:
When JSON will have fully functional comments, this problem will disappear. Until then, when a field starts with `"//"`, it's a comment

Reply:
@vbauzysvmware would it help if it was formatted inside differently and a soft language were used, so you don't want to uncomment, for example:
```
    "// ": "vcd.org -> Organization name",
    "// ": "vcd.vdc -> VDC name",
    "// ": "vcd.catalog.name -> Catalog name",
    ...
```

Reply:
Yep, looks better

Reply:
changed